test/e2e/azure/cli.go

```diff
@@ -77,26 +80,50 @@ func NewAccount() (*Account, error) {
 		return nil, err
 	}
 	a.User = u
+	a.StorageAccount = new(StorageAccount)
+
+	cmd := exec.Command("which", "timeout")
```

Comment:
are we expecting to see instances where this isn't true?

Reply:
my Mac doesn't have it :)